### ISSUE TITLE
🚀  Outline none should be replaced by outline transparent for accessibility reasons

### ISSUE THREAD

**Issue by fraincs** [gsoft-inc/sg-orbit#983]
## Description

See https://twitter.com/alastc/status/1252213068542676992
